### PR TITLE
Make DOCKER_TAGNAME optional in modules Makefile and update mover val…

### PR DESCRIPTION
**Pull request by revit13**
This PR does the following:

1) Make DOCKER_TAGNAME setting optional in modules Makefile 
2) update mover tag in values.yaml file upon release

### CODE REVIEW

File: modules/fybrik-implicit-copy-batch/Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ all: helm-all
 
 ROOT_DIR := ../..
 DOCKER_NAME := fybrik-implicit-copy-batch
-DOCKER_TAGNAME := 0.1.0
+DOCKER_TAGNAME ?= 0.1.0
```
Comment (roytman) [inline modules/fybrik-implicit-copy-batch/Makefile]: should we use the default tag as 0.0.0 ? as in the fybrik case

File: modules/fybrik-implicit-copy-stream/Makefile [hunk 2]
```diff
@@ -3,7 +3,7 @@ all: helm-all
 
 ROOT_DIR := ../..
 DOCKER_NAME := fybrik-implicit-copy-stream
-DOCKER_TAGNAME := 0.1.0
+DOCKER_TAGNAME ?= 0.1.0
```
Comment (roytman) [inline modules/fybrik-implicit-copy-stream/Makefile]: as above.

File: modules/fybrik-implicit-copy-batch/Makefile (hunk 1 above)
Comment (revit13) [reply modules/fybrik-implicit-copy-batch/Makefile]: @roytman in this version I think it should be `master` - this is the tag for the dev images. In the support for helm 3.7  that should be `0.0.0`

Comment (roytman) [reply modules/fybrik-implicit-copy-batch/Makefile]: but you use `0.1.0` instead of `master`

**Review by roee88 — APPROVED** [review]

**Pull request merged**